app/components/dashboard/collection_component.rb

```diff
@@ -1,12 +1,12 @@
-# typed: strict
+# typed: false
 # frozen_string_literal: true
 
 module Dashboard
   # Renders a collection and a summary table of works in the collection
   class CollectionComponent < ApplicationComponent
     MAX_DEPOSITS_TO_SHOW = 4
 
-    sig { params(collection: Collection).void }
+    # sig { params(collection: Collection).void }
```

Comment:
This is incompatible with the check that view_component does to see if the initializer accepts a kwarg `collection`